common/remote/sockfile.cpp

```diff
@@ -7146,11 +7199,74 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                     }
                 }
 
+                if (rowServiceSockAvail)
+                {
+                    Owned<ISecureSocket> ssock;
+                    try
+                    {
+                        acceptedRSSock.setown(rowServiceSock->accept(true));
+                        if (!acceptedRSSock||stopping)
+                            break;
+
+                        if ((rs_onssl == rowServiceCfg) || (rs_bothssl == rowServiceCfg)) // NB: will be disabled if !_USE_OPENSLL
+                        {
+                            ssock.setown(createSecureSocket(acceptedRSSock.getClear(), ServerSocket));
+                            int status = ssock->secure_accept();
+                            if (status < 0)
+                                throw createDafsException(DAFSERR_serveraccept_failed,"Failure to establish SSL row service connection");
+                            acceptedRSSock.setown(ssock.getLink());
+                        }
+                    }
+                    catch (IJSOCK_Exception *e)
+                    {
+                        // accept failed ...
+                        EXCLOG(e,"CRemoteFileServer (row service)");
+                        e->Release();
+                        break;
+                    }
+                    catch (IException *e) // IDAFS_Exception also ...
+                    {
+                        EXCLOG(e,"CRemoteFileServer1 (row service)");
+                        e->Release();
+                        cleanupSocket(acceptedRSSock);
+                        sockSSL.clear();
+                        cleanupSocket(ssock);
+                        ssock.clear();
+                        rowServiceSockAvail = false;
+                    }
+                }
+
+                SocketEndpoint eps;
+                StringBuffer sb;
```

Comment:
sb could be more self documenting peerURL?

Reply:
not new, just moved, but can change.